docs/workflows/authentication.rst

```diff
@@ -0,0 +1,75 @@
+.. _authentication:
+
+Registry Token Authentication
+=============================
+
+Pulp has a support for pulling a content from an **authenticated registry**. This enables users
+to pull the content with an authorized access. A token server grants such an access based on the
+user's privileges and current scope. Note that a role based access control, as a separate feature,
+is considered to be implemented in the future.
+
+The token authentication is **disabled by default**. To enable this feature, it is necessary to
+set up a fully qualified domain name for the token server in the file ``pulp_docker/app/settings.py``.
+Append the variables ``TOKEN_SERVER`` and ``TOKEN_SIGNATURE_ALGORITHM`` to the settings. Do not forget
+to set a valid path for a private key and for a public key. These keys are used to sign and validate
+the signature.
+
+.. code-block:: python
+
+    TOKEN_SERVER = "localhost:24816/token"
+    TOKEN_SIGNATURE_ALGORITHM = 'ES512'
+    PUBLIC_KEY_PATH = '/tmp/public.pem'
+    PRIVATE_KEY_PATH = '/tmp/private.pem'
+
+To learn more about Pulp settings, take a look at `Configuration
+<https://docs.pulpproject.org/en/3.0/nightly/installation/configuration.html>`_.
+
+Run the command ``prestart`` in order to reload the updated settings. Pulp will fetch a domain for
```

Comment:
`prestart` is only available in the development environment. For users, you can just say "restart the services".